File: app/src/main/java/com/twt/zhihu/zhihudaily/Presenter/MainContract.java
package com.twt.zhihu.zhihudaily.Presenter;

import com.twt.zhihu.zhihudaily.Model.MainBean;

public class MainContract {
    public interface UIview {

        void initView();
        void updateView();
        void setData(MainBean mainBean);
    }

    public interface ListPresenter{
        void getList();
        void getLoad(String date);
    }
}

File: app/src/main/java/com/twt/zhihu/zhihudaily/presenter/MainContract.kt
package com.twt.zhihu.zhihudaily.presenter

import com.twt.zhihu.zhihudaily.model.MainBean

class MainContract {
    interface UIview {
        fun initView()
        fun updateView()
        fun firstInitView()
        fun setData(mainBean: MainBean)
    }

    interface ListPresenter {
        fun getList()
        fun getLoad(date: String)
        fun getInit()
    }
}
